.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Build and Release
+
+on:
+  push:
+    tags:
+      - '*.*'
```

Comment:
so Build and Release will fire for any tags that have a `.` in them?

Reply:
Yeah - I read some stuff that you need a wildcard, and '*' was not working with tags.